flag/service/workload/tenant.go

```diff
@@ -0,0 +1,11 @@
+package workload
```

Comment:
Maybe this file should also be named `workload.go`

Reply:
Thanks, I searched in files but didn't look at filenames. This is fixed now.